server/src/main/java/org/opensearch/action/admin/indices/create/CreateIndexRequestBuilder.java

```diff
@@ -115,6 +115,7 @@ public CreateIndexRequestBuilder setSettings(Map<String, ?> source) {
      * @param source The mapping source
      * @param xContentType The content type of the source
      */
+    @Deprecated
```

Comment:
Same as above

Reply:
same as above...  it's currently a big blocker for removing the type from the translog.. :/